docs/pages/admin-guides/infrastructure-as-code/terraform-provider/long-lived-credentials.mdx

```diff
@@ -30,6 +30,8 @@ which is an additional security layer that protects Teleport in case of Identity
 
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
+- A macOS or Linux host for the Terraform command executions.
```

Comment:
In the previous files, we always said "Linux or macOS". Might as well be consistent.
```suggestion
- A Linux or macOS host for the Terraform command executions.
```